### DISCUSSION TITLE
No Transmission Towers in Landscape Mountains Unreal Environment

### DISCUSSION THREAD

**Discussion by edwardjan** [microsoft/AirSim#3357] [Support Q&A] [answered]
### What's your question?

I've been following the provided tutorial for RL with Quadrotor in https://microsoft.github.io/AirSim/reinforcement_learning/.
I proceeded to download and create a project in using the Landscape Mountains Unreal Environment.

However, the Unreal Environment doesn't seem to have transmission towers. Do they have to be added in? Or is there an environment with them available?

### Include context on what you are trying to achieve

I need to run the provided source code for RL with Quadrotor, so that I can see it in action personally.

#### Context details

OS: Windows 10
AirSim Version: 1.3.1
Python: 3.7.9
Unreal Version: 4.25.4

Setings.json:
{
  "SeeDocsAt": "https://github.com/Microsoft/AirSim/blob/master/docs/settings_json.md",
  "SettingsVersion": 1.2,
  "SimMode": "Multirotor",
  "ClockSpeed": 1.0,
  "Vehicles": {
    "SimpleFlight": {
      "VehicleType": "SimpleFlight",
      "DefaultVehicleState": "Armed",
      "EnableCollisionPassthrogh": false,
      "EnableCollisions": true,
      "AllowAPIAlways": true,
      "RC": {
        "RemoteControlID": 0,
        "AllowAPIWhenDisconnected": false
      }
    }
  }
}

**Comment by jonyMarino** [comment, marked as answer]
Hi @edwardjan! and Welcome! The Landscape Mountains environment in 1.3.1 doesn't have power lines. But the one in 1.2 has it. So, try with the binary from that version and tell us if you get another issue (you will not be able to use the new API methods that were added since 1.2)

**Reply by puhuofeie11** [reply]
excuse me,how to find the  Landscape Mountains environment in 1.2? i can't  find it in ue  market.

**Comment by edwardjan** [comment]
Thank you for the response. I was able to get the drone up and running. For any other people with the same issue, you will need Python 3.6 since CNTK is needed and it can't be installed in Python 3.7 and onwards.

I think we can close this query, thanks again.

**Reply by iibuan** [reply]
This is really weird because it mentions here `https://microsoft.github.io/AirSim/reinforcement_learning/` that the tutorial works with environment available in releases.
The tutorial they have on the documentation uses stable baselines3, which requires python 3.7. 
What is the use of the tutorial if there is no compatible environment you can use it for?

**Comment by IuliuNovac** [comment]
Hi, is the Landscape Mountains environment with transmission lines from the 1.2 binaries available somewhere? I would like to use it with the latest version of AirSim.

**Comment by DuWeiMing** [comment]
You can download the&nbsp;&nbsp;Landscape Mountains&nbsp;map from https://github.com/microsoft/AirSim/releases/tag/v1.2.0.&nbsp;&nbsp;
But&nbsp;you will not be able to use the new API methods that were added since 1.2.







------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Iuliu ***@***.***&gt;; 
发送时间: 2021年11月17日(星期三) 晚上9:00
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [microsoft/AirSim] No Transmission Towers in Landscape Mountains Unreal Environment (Discussion #3357)





 
Hi, is the Landscape Mountains environment with transmission lines from the 1.2 binaries available somewhere? I would like to use it with the latest version of AirSim.
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
